src/platform/Linux/ConfigurationManagerImpl.cpp

```diff
@@ -369,17 +369,19 @@ CHIP_ERROR ConfigurationManagerImpl::StoreBootReason(uint32_t bootReason)
 
 CHIP_ERROR ConfigurationManagerImpl::GetRegulatoryLocation(uint8_t & location)
 {
-    uint32_t value = 0;
-
-    CHIP_ERROR err = ReadConfigValue(PosixConfig::kConfigKey_RegulatoryLocation, value);
+    uint32_t value;
 
-    if (err == CHIP_NO_ERROR)
+    if (CHIP_NO_ERROR != ReadConfigValue(ConfigClass::kConfigKey_RegulatoryLocation, value))
+    {
+        ReturnErrorOnFailure(GetLocationCapability(location));
+        StoreRegulatoryLocation(location);
```

Comment:
StoreRegulatoryLocation seems to return a CHIP_ERROR. Should we also error check (and error return or log) if that fails?

Reply:
I prefer logging an error, since we should not return error as long as we can get the default value from GetLocationCapability